doc/getting-started/understanding-config-files.md

```diff
@@ -19,7 +19,7 @@ Tells your node to which nodes in the network it should talk to. A minimal versi
 
 * `valency` tells the node how many connections your node should have. It only has an effect for dns addresses. If a dns address is given, valency governs to how many resolved ip addresses should we maintain active (hot) connection; for ip addresses, valency is used as a Boolean value, where `0` means to ignore the address.
 
-Your __block-producing__ node must __ONLY__ talk to your __relay nodes__, and the relay node should talk to other relay nodes in the network. Go to our telegram channel to find out IP addresses and ports of peers.
+Your __block-producing__ node must __ONLY__ talk to your __relay nodes__, and the relay nodes should talk to other relay nodes in the network. Go to https://explorer.cardano.org/relays/topology.json to find out IP addresses and ports of peers.
```

Comment:
The link would not show up new relays immediately afaik, last I checked - @disassembler mentioned it was refreshed twice a week, but that's almost a year ago.
So if mentioning the link and above is true, would be good to put a clause around the update window for new folks (as the guide is predominantly targetting those who might not be familiar with the configs or references).

Reply:
Update to include that the link is updated once a week.